sdccc/src/main/java/com/draeger/medical/sdccc/messages/MessageStorage.java

```diff
@@ -355,10 +362,14 @@ protected Charset determineCharsetFromMessage(final Message message) {
                 } else if (charset.equals(charsetFromXmlDeclaration)) {
                     charsetWithOrigin = String.format("'%s' in XML Declaration", charset);
                 }
-                this.testRunObserver.invalidateTestRun(String.format(
-                        "Encountered a message whose encoding is declared to be %s. This violates"
-                                + " MDPWS:R0007_0 - SOAP ENVELOPEs SHALL be encoded by using UTF-8.",
-                        charsetWithOrigin));
+                if (this.summarizeMessageEncodingErrors) {
```

Comment:
The test run will not be invalidated if SummarizeMessageEncodingErrors is set.

Reply:
Redesigned it. Now there is a separate Configuration Key for that.